### PR TITLE
Fixes ranged guardian light

### LINKED ISSUE #14661: Holoparasite lights do not stay with you.
If you toggle on your light as a holoparasite, then move, the light will stay where you turned it on, rather than move with you.

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
And by fix I mean removed it and replaces it with toggleable night vision, because I'm not going to to rewrite incorporeal move.

Fixes #14661

### CODE REVIEW

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -436,6 +436,8 @@
 	playstyle_string = "As a ranged type, you have only light damage resistance, but are capable of spraying shards of crystal at incredibly high speed. You can also deploy surveillance snares to monitor enemy movement. Finally, you can switch to scout mode, in which you can't attack, but can move without limit."
 	magic_fluff_string = "..And draw the Sentinel, an alien master of ranged combat."
 	tech_fluff_string = "Boot sequence complete. Ranged combat modules active. Holoparasite swarm online."
+	see_invisible = SEE_INVISIBLE_MINIMUM
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Won't this mean it'll always have no darkness overlay?

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Guess I should toggle that instead of see_in_dark

**Comment by KorPhaeron** [comment]
@tkdrg 

Ready now

**Comment by KorPhaeron** [comment]
@tkdrg 

For real

**Pull request merged**